eng/tools/tasks/Microsoft.DotNet.UnifiedBuild.MSBuildSdkResolver/Microsoft.DotNet.UnifiedBuild.MSBuildSdkResolver.csproj

```diff
@@ -6,8 +6,8 @@
 
   <ItemGroup>
     <!-- IncludeAssets=compile to treat these packages as targeting-packs only. The assemblies are available in the SDK. -->
-    <PackageReference Include="Microsoft.Build.Tasks.Core" IncludeAssets="compile" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" IncludeAssets="compile" />
+    <PackageReference Include="Microsoft.Build.Tasks.Core" IncludeAssets="compile" NoWarn="NU1901;NU1902;NU1903;NU1904" />
```

Comment:
nit: It'd be nice to have a shared ItemGroup to reference here for the NuGet warnings named something like `@(NuGetAuditWarning)`.

Reply:
I think to be usable here it'd need to be a property.

Reply:
Agreed. This best belongs directly into nuget's NuGet.props so that customers benefit from the list as well: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

@zivkan @nkolev92 any objections on that?

Reply:
There's a lot of movement around audit, so I'd like for it to be considered in the scope of that work as a potential enhancement.

fwiw, suppressing warnings, I'd almost rather this be more difficult than easier for people to do.

Reply:
We also have other features in mind that should eventually address the auditing of packages that we don't deploy like Microsoft.Build.Tasks.Core. 

https://github.com/nuget/home/issues/13860 might be related as well.